pkg/rego/schemas/cloud.json

```diff
@@ -502,6 +506,147 @@
         }
       }
     },
+    "github.com.aquasecurity.defsec.pkg.providers.aws.autoscaling.Autoscaling": {
+      "type": "object",
+      "properties": {
+        "autoscalinggroupslist": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "$ref": "#/definitions/github.com.aquasecurity.defsec.pkg.providers.aws.autoscaling.AutoscalingGroupsList"
+          }
+        },
+        "launchconfigurations": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "$ref": "#/definitions/github.com.aquasecurity.defsec.pkg.providers.aws.autoscaling.LaunchConfigurations"
+          }
+        },
+        "notificationconfigurations": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "$ref": "#/definitions/github.com.aquasecurity.defsec.pkg.providers.aws.autoscaling.NotificationConfigurations"
+          }
+        }
+      }
+    },
+    "github.com.aquasecurity.defsec.pkg.providers.aws.autoscaling.AutoscalingGroupsList": {
+      "type": "object",
+      "properties": {
+        "autoscalinggrouparn": {
+          "type": "object",
+          "$ref": "#/definitions/github.com.aquasecurity.defsec.pkg.types.StringValue"
+        },
+        "avaiabilityzone": {
```

Comment:
probably need to regen the schema once you've fixed the typo

Reply:
yes. I will do it